starter-code/styles/base.css

```diff
@@ -1,4 +1,8 @@
 /* TODO: Add a rule for images, so they never exceed their container's width. */
+img{
+  max-width: 100%;
+  height: auto;
```

Comment:
The `height` and `width` sizes are set to auto by default so you shouldn't need to set it here. The `max-width` is spot on though!